### PR TITLE
Adding toolbar and headline

### PR DESCRIPTION
**Pull request by shadabwahidullah**
I have added the toolbar and headline

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,8 +6,36 @@
...
-    
+<body class="container">
+    <div class="toolbar">
+        <h2 class="logo">Wahid Shadab</h2>
+        <div class="menu">
+            <div class="menu_item"></div>
+            <div class="menu_item"></div>
+            <div class="menu_item"></div>
+        </div>
+    </div>
```
Comment (carlos-ssh) [inline index.html:22]: - The container of the section is a header tag.
- The logo (if the design has one) is on the left, and has a link (a tag).
- The hamburger button is on the right, and has a nav tag.

File: index.html [hunk 2]
```diff
@@ -6,8 +6,36 @@
...
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
+    <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+
 </head>
-<body>
-    
+<body class="container">
```
Comment (carlos-ssh) [inline index.html:14]: - Kindly fix white spaces: 
Screen Shot 2021-08-16 at 19 04 08

File: index.html [hunk 3]
```diff
@@ -6,8 +6,36 @@
...
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="social_media">
+            <div class="fa fa-linkedin fa-2x"></div>
+            <div class="fa fa-github fa-2x"></div>
+            <div class="fa fa-twitter fa-2x"></div>
+            <div class="fa fa-facebook fa-2x"></div>
+            <div class="fa fa-youtube fa-2x"></div>            
+        </div>
```
Comment (carlos-ssh) [inline index.html:38]: - The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @shadabwahidullah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -6,8 +6,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
[the remark this answers is not in the document]
Comment (shadabwahidullah) [reply index.html:6]: I could not understand what to do with the meta tag?
Can you give some more info.
I have already update the readme file, Is there anything I have to do with the meta tag?

**Review by george-shammar — APPROVED** [review]
Hi @shadabwahidullah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**